src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -149,26 +156,23 @@ class WalletService(
                             )
                     )
                     .map { hostedOrderResponse ->
-                        Triple(hostedOrderResponse, wallet, Pair(orderId, contractId))
+                        SessionCreationData(
+                            hostedOrderResponse,
+                            wallet.copy(
+                                contractId = ContractId(contractId),
+                                status = WalletStatusDto.INITIALIZED
```

Comment:
wallet status update must be differentiate from cards and apm onboardings.
for cards -> `INITIALIZED`
for apm -> `VALIDATION_REQUESTED` since for other methods flow will go directly to external domain redirection, right?
Also, for paypal case here we should call NPG with paypal specific api key.